dev/releases/HelpLinks-to-JSON.g

```diff
@@ -0,0 +1,53 @@
+##  
+##  A utility for dumping links to all available manual sections 
+##  (for use with named manual references on web pages or in search
+##  utilities).
+##
+##  This is meant to be used with a plain vanilla GAP release, with
+##  only the bundled packages and no special user config.
+##
+
+LoadPackage("json");
+
+InstallMethod(_GapToJsonStreamInternal,
+[IsOutputStream, IsObject],
+function(o, x)
+    PrintTo(o, "null");
+end);
+
+out := OutputTextUser();
+r := rec();
+# load all books (without prompting the user for input)
+Perform(HELP_KNOWN_BOOKS[1], HELP_BOOK_INFO);;
```

Comment:
I guess the point is to ensure the list is sorted?

Reply:
Ah, I presume you mean for the line below? I was confused for a minute. I'll comment about my thinking on that line specific line.